gpflow/models/model.py

```diff
@@ -103,6 +104,32 @@ def __init__(
         self.kernel = kernel
         self.likelihood = likelihood
 
+    @staticmethod
+    def num_latent_gps_for_likelihood_and_data(likelihood, data) -> int:
+        _, Y = data
+        output_dim = Y.shape[1]
+        return GPModel.num_latent_gps_for_likelihood_and_output_dim(likelihood, output_dim)
+
+    @staticmethod
+    def num_latent_gps_for_likelihood_and_output_dim(likelihood, output_dim) -> int:
+        """
+        Note: It's not nice for `GPModel` to need to be aware of specific
+        likelihoods as here. However, `num_latent_gps` is a bit more broken in
+        general, specifically regarding multioutput kernels. We should fix this
```

Comment:
check that we've got a separate issue to keep track of this; add issue url to code

Reply:
@condnsdmatters I merged your suggestion in here. Do you think we can remove this point from the docstring now ?